### ISSUE TITLE
[BUG] Custom FBW, axis not being recognized. 

### ISSUE THREAD

**Issue by JuanWhite** [flybywiresim/a32nx#3203]
**Mod Version**
Unstable Master Branch Build (with custom FBW).
```
{
  "built": "2021-01-26T12:52:29+00:00",
  "ref": "",
  "sha": "",
  "actor": "Andreas Guther <andreas@guther.family>",
  "event_name": "manual"
} 
```

**Describe the bug**
I use this version as it is the only way to have my TCA Airbus Quadrant to work properly. It used to work perfectly following this [instructions](https://github.com/flybywiresim/a32nx/tree/fbw/docs) but today I downloaded the latest version from the server as usual and once on the cockpit none of the throttle, joystick or rudder axis move thy are blocked on the central position and cannot be moved either manually using the mouse or other controllers aside from my TCA joystick and quadrant . Sadly I deleted the version that worked and cannot roll back. trying to fix it I discovered that the axis worked after deleting all mod files from both community folder and C:\Users\peque\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages. But they only work for the first time you load the aircraft. Then the ThrottleConfiguration.ini and other files are created. Reload the plane and axis are blocked again. I think there is something wrong with the files created on the local state folder. I really hope you can help me, I spent almost half a day trying to fix this. Sorry for my non-native English.

**To Reproduce**

1. Use the Custom FBW for the first time to have all your files on the Local State folder created, axis should work on this first load with a clean install.
2. Reload the aircraft restarting the sim or using the dev console. Now the axis are blocked.
3.

**Expected behavior**

Being able to use the axis I mentioned above, and modifying the ThrottleConfiguration.ini so I can use my TCA quadrant . It worked on the version I downloaded on the 25/01/2021

**Actual behavior**

Axis are blocked if mod files are present on the LocalState folder

**References**
[VIDEO](https://youtu.be/7eq8Ek4j6M4)

**Additional context**

I have my community folder empty but the A320NX

Discord username (if different from GitHub): Juan White#1670

**Comment by JuanWhite** [comment]
PLEASE READ!
Sorry, for some reason the **describe the bug** section is not showing up.
I leave it on this comment.
**Describe the bug**

**Comment by MisterChocker** [comment]
cc. @aguther

**Comment by aguther** [comment]
One thing I've seen is that you set the full reverse to -0.9 and idle reverse to -1.0, which does not make sense. I guess you mean it the way around. I assume this causes issues in the code. Can you check the output of the console, using the filter "WASM"?

**Comment by JuanWhite** [comment]
> I just tried it on my end and it works without issues for me, also with latest version.
> 
> One thing I've seen in your video is that you set the full reverse to -0.9 and idle reverse to -1.0, which does not make sense. I guess you mean it the way around. I assume this causes issues in the code. Can you check the output of the console, using the filter "WASM" and also to fix the mapping.

Thanks for the answer!
About the settings you saw on the video I was following the instructions to config the TCA a320 Quadrant as shown [here](https://github.com/flybywiresim/a32nx/tree/fbw/docs). But I think that is not the problem. As I get the same issue with a default ThrottleConfiguration.ini file. I´ve just tried with most recent version and same thing. With a clean install of the mod all axis move but restart the flight and all axis are blocked again.
About the WASM filter on the console I am not sure how to use it but I will try and let you know.

**Comment by aguther** [comment]
Well, the doc states it in the way I described it:
DetentReverseFull = -1.00
DetentReverseIdle = -0.90

you had:
DetentReverseFull = -0.90
DetentReverseIdle = -1.00

The screenshot is what I was asking for. Was this done with the default config or with the config you‘ve shown in the video?

**Comment by aguther** [comment]
Just did a recheck of the code and also tried with wrong configuration. I have no idea what is going on, but I found a new interesting point. I'm missing the message "SimConnect connection established" in your screenshot. Here is how it looks like for me:
![image](https://user-images.githubusercontent.com/6629914/106560781-dc5acc00-6527-11eb-998e-de5b66d3aac9.png)

So the question is why is that message not coming? It seems it fails to connect to SimConnect, but why are then your controls already masked (which happens over SimConnect)? Does this message come when it works?

Do you have other add-ons like FSUIPC? Do they have issues connecting?

**Comment by JuanWhite** [comment]
Screenshot was using the default. I will record a video following the steps you described.
I have FSUIPC and it seems to work fine.
Hope we can get to the bottom of this. Thanks for the effort!

**Comment by JuanWhite** [comment]
Ok, done the video. 
Something weird happened this time. After a clean install, axis worked then restarted and they still worked. Then tried to change ThrottleConfiguration.ini following the instructions provided. Restarted again and axis blocked. Restarted one more time deleting the ThrottleConfiguration.ini so I get a default one. Axis cannot be moved either. 
Last time I tried I got the blocked axis on the first restart.
 
Please take the time to see on my horrible video.
Thanks!

https://youtu.be/SS1icKlz8qk

**Comment by aguther** [comment]
Well, still the same issue with the setting.

I don't get why you're setting reverse idle to a lower value than full reverse. Let me explain what that would mean in the real aircraft: it would mean, you release the reverse lockers and it goes to full reverse thrust and then you put them full aft and it goes to reverse idle.

Whatever values you use, please increase them in that order, starting with the smallest number (usually -1.00) and ending with the largest (usually 1.00):
```
DetentReverseFull
DetentReverseIdle
DetentIdle
DetentClimb
DetentFlexMct
DetentTakeOffGoAround
```

That is how it's meant to be:
![image](https://user-images.githubusercontent.com/6629914/106780880-b842dc00-6648-11eb-8ae7-028f9f0dc09d.png)

That is how you configure it:
![image](https://user-images.githubusercontent.com/6629914/106781087-f4763c80-6648-11eb-99e4-cf447511ec35.png)

Please try it with the following setting:
```
[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.5
DetentReverseFull = -1.00
DetentReverseIdle = -0.90
DetentIdle = -0.42
DetentClimb = 0.06
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00
```

**Comment by JuanWhite** [comment]
Tried with the configuration yo sent me. Still same issue, please take a look to the video.
Thanks for your time (;

https://youtu.be/rx6Mntf4uaQ

**Comment by aguther** [comment]
This issue is really stubborn...hmm. Can you please try to edit the file as you need and then restart the sim? I want to know if it's something like it works in principle and your config or saving of file or whatever is not the reason. Maybe something happens on your end when reloading the flight.

**Comment by JuanWhite** [comment]
Also tried to reload the sim with both default and modified file when I finished recording. Same issue, axis blocked. I can record it if you wanna see the console.

**Comment by aguther** [comment]
meaning it's happening right away after loading the sim, not only after reloading the plane?

**Comment by JuanWhite** [comment]
The only time axis work, is with a fresh install of the mod, after deleting the A320nx folder form %appdata%-local-packages etc...
I only works for the first time the plane spawns. Any subsequent reloads or sim restarts will result on the same issue. It does not matter if the first reload is with a default or modified ThrottleConfiguration.ini file.

**Comment by aguther** [comment]
Ok, I will add some more logging, maybe we find more about it with that. I inform you when it's up.

**Comment by JuanWhite** [comment]
Sorry for the late reply, missed the mail notification.
I am glad to inform you that I just downloaded the last custom version ("built": "2021-02-09T23:20:08+00:00") and everything is back to normal, just as before I started this issue report. I am using the following settings:
[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.5
DetentReverseFull = -1.00
DetentReverseIdle = -0.90
DetentIdle = -0.42
DetentClimb = -0.01
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

So I guess this is solved for good.
Thanks one more time for your work and effort.
Have a great week (;
